MIGRATION.md

```diff
@@ -2,7 +2,7 @@
 
 ## 3.9.x to 3.10.0
 
-- Web support is no longer included in default export and is now a separate export `RadarRNWeb`.
+- Web support is no longer included in default export and is now a separate export `RadarRNWeb`. Call `import {RadarRNWeb} from Radar`
```

Comment:
`import { RadarRNWeb } from 'react-native-radar'` (with spaces).

Importing both would be `import Radar, { RadarRNWeb } from 'react-native-radar'`